src/components/pages/main-page/MainPage.tsx

```diff
@@ -38,114 +39,115 @@ const MainPage: FC<MainPageProps> = ({ data }) => {
-                          disabled={false}
-                          color={ButtonColor.PRIMARY}
-                          variant={ButtonVariant.FILLED}
-                          size={ButtonSize.LARGE}
-                        />
-                      </Link>
-                      <hr className={styles['button-divider']} />
-                    </>
-                  )}
-                  <Link href={'/priority'}>
-                    <Button
-                      text={'Обрати пріоритет'}
-                      disabled={false}
-                      variant={ButtonVariant.OUTLINE}
-                      size={ButtonSize.LARGE}
-                    />
-                  </Link>
-                </div>
-                <div className={styles['buttons-tabl']}>
-                  {!isLoggedIn && (
-                    <>
-                      <Link href={'/contract'}>
-                        <Button
-                          text="Договір про навчання"
-                          disabled={false}
-                          color={ButtonColor.PRIMARY}
-                          variant={ButtonVariant.FILLED}
-                          size={ButtonSize.MEDIUM}
-                        />
-                      </Link>
-                      <hr className={styles['button-divider']} />
-                    </>
-                  )}
-                  <Link href={'/priority'}>
-                    <Button
-                      text={'Обрати пріоритет'}
-                      disabled={false}
-                      variant={ButtonVariant.OUTLINE}
-                      size={ButtonSize.MEDIUM}
-                    />
-                  </Link>
-                </div>
-                <div className={styles['buttons-mob']}>
-                  {!isLoggedIn && (
-                    <>
-                      <Link href={'/contract'}>
-                        <Button
-                          text="Договір про навчання"
-                          disabled={false}
-                          color={ButtonColor.PRIMARY}
-                          variant={ButtonVariant.FILLED}
-                          size={ButtonSize.SMALL}
-                        />
-                      </Link>
-                    </>
-                  )}
-                  <Link href={'/priority'}>
-                    <Button
-                      text={'Обрати пріоритет'}
-                      disabled={false}
-                      variant={ButtonVariant.OUTLINE}
-                      size={ButtonSize.SMALL}
-                    />
-                  </Link>
-                </div>
-              </div>
-            </div>
-          </div>
-          <div
-            className={cn(styles['build-image'], {
-              [styles['animate']]: !isIOS && !isSafari,
-            })}
-          >
+              </Typography>
+              <Box sx={stylesMUI.buttonDesk}>
+                {!isLoggedIn && (
+                  <>
```

Comment:
you don't need to wrap Link in Fragment

Reply:
TS2657: JSX expressions must have one parent element. That`s why Fragment should stay for buttonDesk version (it has 2 children in the login-check)

Reply:
I left it dynamic version of buttons for the same reason